cfpb/cfgov-refresh
# Remove feature flags from mega menu for things that have launched

Remove feature flags for mega menu items that have been released (hold until after Debt Collection release on 1/12)

## Additions

- Add feature-flagged items directly to `nav_items`  and `media_items` in  `_vars-mega-menu.html`

## Removals

- Remove feature flag conditions from  `_vars-mega-menu.html`

## Testing

- Check out branch
- Verify that megamenu links still appear & work as expected: `Prepaid Cards` & `Debt Collection` links under `Consumer Tools`, and `Consumer Credit Trends` under `Data & Research`
- Verify that debt collection featured menu content appears under `Consumer Tools`

## Screenshots
<img width="1134" alt="screen shot 2017-01-11 at 8 53 20 pm" src="https://cloud.githubusercontent.com/assets/778171/21877461/21e6066e-d840-11e6-9390-dd1f5d46c5ef.png">

<img width="954" alt="screen shot 2017-01-11 at 8 04 35 pm" src="https://cloud.githubusercontent.com/assets/778171/21876691/295fd27c-d83a-11e6-8ba6-6835982780e9.png">

## Checklist
